libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -56,6 +57,43 @@ describe('ModalWrapperComponent', () => {
     });
   });
 
+  describe('sizing', () => {
+    beforeEach(() => {
+      spectator = modalWrapperTestBuilder
+        .flavor('modal')
+        .withEmbeddedInputComponent()
+        .build();
+    });
+    afterEach(() => {
+      spectator.fixture.destroy();
+    });
+
+    it('should trigger `onScrollElementResize` when embeddded component resizes', (done) => {
+      setTimeout(() => {
+        // @ts-ignore
+        expect(spectator.component.resizeObserverService.observe).toHaveBeenCalledWith(
+          // @ts-ignore
```

Comment:
Why this? If you need to access private members you can do the following, which still gets type checked:
```suggestion
        expect(spectator.component['resizeObserverService'].observe).toHaveBeenCalledWith(
```